docs/deployed-contracts/holesky.md

```diff
@@ -82,6 +82,7 @@
 - CSFeeDistributor: [`0xD7ba648C8F72669C6aE649648B516ec03D07c8ED`](https://holesky.etherscan.io/address/0xD7ba648C8F72669C6aE649648B516ec03D07c8ED) (proxy)
 - CSFeeOracle: [`0xaF57326C7d513085051b50912D51809ECC5d98Ee`](https://holesky.etherscan.io/address/0xaF57326C7d513085051b50912D51809ECC5d98Ee) (proxy)
 - CSVerifier: [`0x6FDAA094227CF8E1593f9fB9C1b867C1f846F916`](https://holesky.etherscan.io/address/0x6FDAA094227CF8E1593f9fB9C1b867C1f846F916)
```

Comment:
What about this instance of verifier? I believe it's needed to add some tag here  (proposed to remove) as well.